src/components/image-fetch/AdopterImage.tsx

```diff
@@ -0,0 +1,38 @@
+import { graphql, useStaticQuery } from "gatsby";
+import Img from "gatsby-image";
```

Comment:
Since you have already used the new `gatsby-plugin-image` and the old `gatsby-image` is no more required.
This will do auto migration: https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#migrating
After migration, if you feel some images are static like this one use the appropriate `<StaticImage />` component: https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#static-images

Reply:
I've successfully switched over to gatsby-plugin-image